### PR TITLE
Base grenade prototype

### PR DESCRIPTION
**Pull request by AlexMorgan3817**
## About the PR
Grenades got their parent prototype
No C#
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # TODO probably needs a base grenade
-
 - type: entity
-  name: explosive grenade
-  description: Grenade that creates a small but devastating explosion.
+  name: basic grenade
+  description: How did you get it???
```
Comment (moonheart08) [inline Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml:4]:
```suggestion
  abstract: true
```

File: Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
   parent: BaseItem
-  id: ExGrenade
+  id: BasedGrenade
```
Comment (moonheart08) [inline Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml:6]: BaseGrenade, not BasedGrenade. Fix your (intentional, don't do that) typo.

Comment (AlexMorgan3817) [reply Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml:6]:
```suggestion
  id: BaseGrenade
```

**Comment by AlexMorgan3817** [comment]
You know, it is done 🐸

**Review by AJCM-git — APPROVED** [review]
Haven't tested in-game but this looks fine

**Comment by AlexMorgan3817** [comment]
Hm

**Pull request merged**